pallets/ethereum-transactions/src/tests/tests_eth_transaction_type.rs

```diff
@@ -28,21 +33,24 @@ fn generate_publish_root_eth_txn_desc(root_hash: [u8; 32]) -> EthTransactionDesc
         call_values: vec![Token::FixedBytes(root_hash.to_vec())],
     }
 }
-
 fn generate_deregister_validator_eth_txn_desc(
+    t1_public_key: H512,
     t2_public_key: [u8; 32],
 ) -> EthTransactionDescription {
     EthTransactionDescription {
         function_call: Function {
             name: String::from("deregisterValidator"),
-            inputs: vec![Param {
-                name: String::from("_targetT2PublicKey"),
-                kind: ParamType::FixedBytes(32),
-            }],
+            inputs: vec![
+                Param { name: String::from("_targetT2PublicKey"), kind: ParamType::FixedBytes(32) },
+                Param { name: String::from("_targetT1PublicKey"), kind: ParamType::FixedBytes(64) },
+            ],
             outputs: Vec::<Param>::new(),
             constant: false,
         },
-        call_values: vec![Token::FixedBytes(t2_public_key.to_vec())],
+        call_values: vec![
+            Token::FixedBytes(t2_public_key.to_vec()),
+            Token::Bytes(t1_public_key.to_fixed_bytes().to_vec()),
+        ],
```

Comment:
The order needs to be changed. T1 key comes first than T2:
https://github.com/Aventus-Network-Services/avn-bridge/blob/master/contracts/AVNBridge.sol#L381

Reply:
This has been addressed in #165 .